dasharo-compatibility/cpu-cores-count.robot

```diff
@@ -0,0 +1,236 @@
+*** Settings ***
+Library             Collections
+Library             OperatingSystem
+Library             Process
+Library             String
+Library             Telnet    timeout=20 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+# TODO: maybe have a single file to include if we need to include the same
+# stuff in all test cases
+Resource            ../sonoff-rest-api/sonoff-api.robot
+Resource            ../rtectrl-rest-api/rtectrl.robot
+Resource            ../variables.robot
+Resource            ../keywords.robot
+Resource            ../keys.robot
+
+# TODO:
+# - document which setup/teardown keywords to use and what are they doing
+# - go through them and make sure they are doing what the name suggest (not
+# exactly the case right now)
+Suite Setup         Run Keywords
+...                     Prepare Test Suite
+...                     AND
+...                     Skip If    not ${CPU_TESTS_SUPPORT}    CPU tests not supported
+Suite Teardown      Run Keyword
+...                     Log Out And Close Connection
+
+
+*** Test Cases ***
+CCC001.001 CPU Hyper-Threading disabled (Ubuntu)
+    [Documentation]    Check if Hyper-Threading is disabled.
+    Set UEFI Option    HyperThreading    ${FALSE}
+    Power On
```

Comment:
@JanPrusinowski Power On seems redundant, the `Set UEFI Option` kwd does the platform reset